lib/org/openpsa/invoices/interfaces/customer.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @package org.openpsa.invoices
+ * @author CONTENT CONTROL http://www.contentcontrol-berlin.de/
+ * @copyright CONTENT CONTROL http://www.contentcontrol-berlin.de/
+ * @license http://www.gnu.org/licenses/gpl.html GNU General Public License
+ */
+
+/**
+ * Interface for identifying the invoice_dba and salesproject_dba
```

Comment:
I would rather say "for identifying DBA entities which can have both person & organization links" or something in this general direction